Makefile

```diff
@@ -50,9 +54,8 @@ test/JSON-Schema-Test-Suite/tests:
 	git submodule sync --recursive
 	git submodule update --init --recursive
 
-# Would be nice to include elvis to test, but it fails on OTP-18
 .PHONY: test
-test: eunit ct xref dialyzer cover
+test: eunit ct xref dialyzer elvis cover
```

Comment:
Hurrah!

Reply:
I'm not sure I agree with this change. Could we take it out of this PR and discuss it some more? It is not required and I'm also thinking it's incomplete (on the latest OTP we would run elvis twice)

Reply:
3190bec